reportd.spec

```diff
@@ -5,7 +5,7 @@ Version:        0.7.4
 Release:        1%{?dist}
 Summary:        Service reporting org.freedesktop.Problems2 entries
 
-License:        GPLv2+
+License:        LGPL-2.0-or-later
```

Comment:
I think this is probably a typo? LGPL is different from GPL.

Reply:
The previous value was actually incorrect. See the [COPYING file](https://github.com/abrt/reportd/blob/master/COPYING).

Reply:
Oh, good catch. I actually checked the file as well, but somehow didn't connect the dots... :D 

lgtm ;)